sdk/netapp/Azure.ResourceManager.NetApp/src/Custom/Models/NetAppVault.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core;
+using Azure.ResourceManager.Models;
+
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
+namespace Azure.ResourceManager.NetApp.Models
+{
+    public partial class NetAppVault : ResourceData
```

Comment:
I believe these three models are moved here because they are deleted in the new api-version along with the deleted operations.

Could we add `[EditorBrowsable(Never)]` to this class? Maybe also obsolete it because it is only used in an obsolete operation.

Also please copy its previous xml document here as well and we no longer need the warning disable clause